Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -7516,11 +7524,13 @@
 											"listen": "test",
 											"script": {
 												"exec": [
-													"pm.test(\"Status is 201\", () => {\r",
-													"  pm.expect(pm.response.code).equals(201);\r",
-													"});\r",
+													"pm.test(\"Response is  201 or 404\", () => {\r",
+													"    pm.expect(pm.response.code).to.be.oneOf([201, 404]) \r",
+													" });\r",
 													"\r",
-													"pm.environment.set('known:parent:id', pm.response.headers.one('Location').value.split(\"/\").pop());"
+													"if (pm.response.code === 201 ) {\r",
```

Comment:
This comment applies to all the Postman tests. Right now, the tests have been modified so that they won't fail for v.5.0+ of the Ed-Fi Standard by making them tolerant of `404` response statuses, but now "Contact" has no test coverage going forward.

_NOTE: Semalai and I discussed this over Slack and he's going to create a new ticket to address this gap in Postman test coverage so that we can get this blocking feature merged._

Reply:
Created new ticket to handle this https://tracker.ed-fi.org/browse/ODS-5921